Makefile-tests.am

```diff
@@ -287,6 +293,17 @@ tests/%-symlink-stamp: % Makefile
 	ln -sf "$${real_bin}" tests/$*; \
 	touch $@
 
+# See above comment on binding the tests to be either installed or not.
+if ENABLE_INSTALLED_TESTS
+installed_test_scripts = $(_installed_or_uninstalled_test_scripts)
+installed_test_programs = $(_installed_or_uninstalled_test_programs)
+check-local:
+	echo "NOTE: Installed tests are enabled; to run them, make install, then: gnome-desktop-testing-runner -p 0 libostree/"
```

Comment:
It's not a no op - again as I said in the commit message, e.g. `test-symbols.sh` is *always* uninstalled and we want to run it.

Your feedback is valuable and accurate!  I'm testing an updated version to add `--enable-installed-tests=exclusive` for use in CI.